src/LAMMPS_oxDNA.py

```diff
@@ -36,7 +36,7 @@ def quat_to_exyz(myquat):
 
 if __name__ == '__main__':
     if len(sys.argv) < 2:
-        print("Usage is %s lammps_init_file" % sys.argv[0], file=sys.stderr)
+        print("Usage is 'python3 %s lammps_data_file' OR 'python3 %s lammps_data_file lammps_trajectory_file'" % (sys.argv[0], sys.argv[0]), file=sys.stderr)
```

Comment:
Could you remove the `python3` bit and use `lammps_data_file [lammps_trajectory_file]` as the list of arguments? This would uniform this script to all the others in the package

Reply:
Not a problem.

Reply:
Sorry, maybe I wasn't super clear. The line should read `print("Usage is '%s lammps_data_file [lammps_trajectory_file]'" % sys.argv[0], file=sys.stderr)`